packages/components/src/menu/README.md

```diff
@@ -1,344 +1,589 @@
-# `Menu`
+# Menu
```

Comment:
@mirka there's no way currently to mark the component as private in the README, right?

Reply:
Right. I'll figure something out as a follow up.

Reply:
Proposed in #68317

Reply:
https://github.com/WordPress/gutenberg/pull/68317 landed so we'll need to rebase and rerun the command to get the private API warning.